CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## CHANGELOG
 
+### 3.11.1 2021-06-11
+
+* Re-package to fix missing file
+* add "prepublishOnly" script and ignore all build assets (#177)
+* fix: switch "package" to "prepublishOnly"
```

Comment:
This line shouldn't be here, this was just the commit message and not for the CHANGELOG